app/models/govbox/message.rb

```diff
@@ -30,16 +30,10 @@ def self.create_message_with_thread!(govbox_message)
     MessageThread.with_advisory_lock!(govbox_message.correlation_id, transaction: true, timeout_seconds: 10) do
       message = create_message(govbox_message)
 
-      thread_title = if message.metadata["delivery_notification"].present?
-                       message.metadata["delivery_notification"]["consignment"]["subject"]
-                     else
-                       message.title
-                     end
-
       message.thread = govbox_message.box.message_threads.find_or_create_by_merge_uuid!(
         box: govbox_message.box,
         merge_uuid: govbox_message.correlation_id,
-        title: thread_title,
+        title: message.metadata.dig("delivery_notification", "consignment", "subject").presence || message.title,
```

Comment:
Ta premenna tu bola lebo sa to takto tazsie citalo, ale moze ostat teraz uz takto.